torch/_inductor/lowering.py

```diff
@@ -2693,6 +2693,8 @@ def is_aligned(x):
 make_fallback(aten._thnn_fused_lstm_cell, require_dense)
 make_fallback(torch._prims.rng_prims.run_and_save_rng_state)
 make_fallback(torch._prims.rng_prims.run_with_rng_state)
+make_fallback(torch._prims.rng_prims.graphsafe_run_with_rng_state)
```

Comment:
do we need all 3 of these rng_prims ops in the long run? Maybe @anijain2305 would know

Reply:
if we're not using cudagraphs, the other ones are still useful bc they have equivalent rng behavior as eager.